File: services/nest-api/src/quote/quote.controller.ts
import { Controller, Get } from '@nestjs/common';
import { QuoteService } from './quote.service';
import { Quote } from './quote.interface';

@Controller('quote')
export class QuoteController {
  public constructor(private readonly service: QuoteService) { }

  @Get()
  public async getHello(): Promise<Quote> {
    return this.service.getQuote();
  }
}

File: apps/kotlin-app/src/main/kotlin/se/frostkin/kotlinapp/QuoteController.kt
package se.frostkin.kotlinapp

import javafx.fxml.FXML
import javafx.scene.control.Label

class QuoteController {
    @FXML
    private lateinit var quote: Label

    @FXML
    private fun initialize() {
        val service = QuoteService()
        val quote = service.getQuote()
        this.quote.text = quote;
    }
}